### PR TITLE
New version: G3G4X5X6.ultimate-cube version 6.6.11

### PR DESCRIPTION
**Pull request by G3G4X5X6**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80750)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
@microsoft-github-policy-service agree company="G3G4X5X6"

**Comment by vedantmgoyal2009** [comment]
@G3G4X5X6 try without quoting the reply

**Comment by vedantmgoyal2009** [comment]
@itzlevvie can this one be merged?

File: manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: G3G4X5X6.ultimate-cube
+PackageVersion: 6.6.11
+PackageLocale: en-US
+Publisher: g3g4x5x6
```
Comment (ItzLevvie) [inline manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.en-US.yaml:7]:
```suggestion
Publisher: G3G4X5X6
```
This must match Control Panel.

File: manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+PublisherUrl: https://g3g4x5x6.github.io/
+PublisherSupportUrl: https://github.com/G3G4X5X6/ultimate-cube/issues
+# PrivacyUrl: 
+Author: G3G4X5X6
+PackageName: ultimate-cube
```
Comment (ItzLevvie) [inline manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.en-US.yaml:12]:
```suggestion
PackageName: ultimatecube
```
This must match Control Panel.

File: manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+PackageIdentifier: G3G4X5X6.ultimate-cube
+PackageVersion: 6.6.11
+PackageLocale: zh-CN
+Publisher: g3g4x5x6
+PublisherUrl: https://g3g4x5x6.github.io/
+PublisherSupportUrl: https://github.com/G3G4X5X6/ultimate-cube/issues
+# PrivacyUrl: 
+Author: G3G4X5X6
+PackageName: ultimate-cube
```
Comment (ItzLevvie) [inline manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.zh-CN.yaml:12]:
```suggestion
PackageName: ultimatecube
```

File: manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.zh-CN.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.2.0.schema.json
+
+PackageIdentifier: G3G4X5X6.ultimate-cube
+PackageVersion: 6.6.11
+PackageLocale: zh-CN
+Publisher: g3g4x5x6
```
Comment (ItzLevvie) [inline manifests/g/G3G4X5X6/ultimate-cube/6.6.11/G3G4X5X6.ultimate-cube.locale.zh-CN.yaml:7]:
```suggestion
Publisher: G3G4X5X6
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**